esm_runscripts/event_handlers.py

```diff
@@ -0,0 +1,36 @@
+import signal
+import sys
+import os
+
+def signal_listener():
+    """Listens to the signals (eg. CTRL-C press, ...) and calls the handler
+    for that signal"""
+    
+    # 1) handler for the SIGINT (CTRL-C)
+    signal.signal(signal.SIGINT, handle_sigint)
+    
+    # other signal handlers may go here, eg. SIGHUP, SIGSTOP, ...)
```

Comment:
`CTRL-Z + bg` might be nice to launch the job, keep the launch going in the background, and getting your shell back. We might need a way to suppress any sort of printing in the case so you don't get those distributing the rest of your work.